src/nspawn/nspawn.c

```diff
@@ -1029,20 +1031,29 @@ static int parse_argv(int argc, char *argv[]) {
                         assert_not_reached("Unhandled option");
                 }
 
+        if (getenv_bool("SYSTEMD_NSPAWN_SHARE_NS_IPC") > 0)
+                arg_share_ns_ipc = true;
+
+        if (getenv_bool("SYSTEMD_NSPAWN_SHARE_NS_PID") > 0)
+                arg_share_ns_pid = true;
+
+        if (getenv_bool("SYSTEMD_NSPAWN_SHARE_NS_UTS") > 0)
+                arg_share_ns_uts = true;
```

Comment:
I think it would make a ton of sense to actually use the getenv_bool() result as the bool it is, so that it can be used to both mask and add fields to the flag set.